include/rootston/osk.c

```diff
@@ -0,0 +1,53 @@
+#include "osk.h"
+
+#include "input-method-unstable-v1-protocol.h"
+
+static void noop() {}
```

Comment:
Add newlines and such and something to the effect of `/* This space deliberately left blank */`